utils/installer/uninstall.sh

```diff
@@ -55,4 +55,5 @@ function main() {
   rm -f "$LVIM_BIN"
 }
 
+echo "Uninstalled LunarVim!"
```

Comment:
It should be moved inside the main function, otherwise it will always print that, even for `--help` 
```suggestion
  rm -f "$LVIM_BIN"
  echo "Uninstalled LunarVim!"
}

```